docs/edge/monitoring.rst

```diff
@@ -0,0 +1,250 @@
+.. _edge-monitoring:
+
+Monitor CrateDB Edge
+====================
+
+This tutorial introduces ways to set up monitoring of your CrateDB Edge cluster.
+Visualization tool `Grafana`_ is used together with `Prometheus`_ and `Loki`_
+to monitor the cluster performance and browse logs, respectively.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+.. _edge-monitoring-prereqs:
+
+Prerequisites
+-------------
+
+- The CrateDB Edge installation script requires Helm to be present in the system.
+- For a deployment with Grafana, Loki, and Prometheus, we recommend at
+  least 5 CPUs and 5 GiB of memory per node for acceptable performance.
```

Comment:
5 CPUs seems to be a lot for just "monitoring"

Reply:
In GKE I had issues deploying with less. By default, we recommend 3 CPUs afaik so I found it reasonable.